orttraining/orttraining/python/training/ortmodule/_custom_autograd_function_exporter.py

```diff
@@ -37,7 +37,7 @@ def _export_pt_1_10(g, n, *args, **kwargs):
                 "wrap exportable sub-nn.Module's as ORTModule."
             )
         inplace = kwargs["inplace"]
-        training_mode = symbolic_helper._training_mode
+        training_mode = symbolic_helper.check_training_mode
```

Comment:
>check_training_mode [](http://example.com/codeflow?start=40&length=19)

might be better to use this for now?
training_mode = symbolic_helper.GLOBALS.training_mode